### ISSUE TITLE
Issue creation in peer_assign() and peer_return() fails

### ISSUE THREAD

**Issue by thereseanders** [rundel/ghclass#93]
Issue creation in peer_assign() and peer_return() fails with the following error message:

```✖ Failed to create issue 'Assigning review' for repo 'ghclass-test/hw6-review-thereseanders'.
  GitHub API error (422): 422 Unprocessable Entity
  └─ Invalid request.
  
  For 'properties/assignees', "thereseanders" is not an array. For
     'properties/labels', ":pencil: Complete review" is not an array. (See
     https://developer.github.com/v3/issues/#create-an-issue)
```

**Comment by thereseanders** [comment]
@mine-cetinkaya-rundel @rundel are you having this problem as well? 

I was trying to work on a solution but cannot figure out whether it is actually an array issue or a permission issue. The [API documentation](https://developer.github.com/v3/issues/#create-an-issue) now includes the following information: "NOTE: Only users with push access can set assignees for new issues. Assignees are silently dropped otherwise."

**Comment by thereseanders** [comment]
I narrowed the problem down to the following:
- Running the entire code chunk below will cause the error above. 
- Running only ```r res = purrr::safely(github_api_issue_create)(repo, title, body, labels, assignees)``` works perfectly.

```r
  res = purrr::pmap(
    list(repo, title, body, labels, assignees),
    function(repo, title, body, labels, assignees) {
      res = purrr::safely(github_api_issue_create)(
        repo, title, body, labels, assignees
      )

      status_msg(
        res,
        glue::glue("Created issue {usethis::ui_value(title)} for repo {usethis::ui_value(repo)}."),
        glue::glue("Failed to create issue {usethis::ui_value(title)} for repo {usethis::ui_value(repo)}.")
      )

      res
    }
  )
```

**Comment by rundel** [comment]
I think we can close this now, can you confirm @thereseanders ?

**Comment by thereseanders** [comment]
Yes, this can be closed as fixed through PR #99.
